src/pages/MainPage.vue

```diff
@@ -254,6 +267,19 @@ export default {
     };
   },
   computed: {
+    useDark: {
+      get () {
+        return this.$store.state.useDarkTheme;
+      },
+      set (value) {
+        this.$store.commit('setUseDarkTheme', value);
+        this.$vuetify.theme.primary = value ? this.$vuetify.theme.discordOrange : this.$vuetify.theme.defaultPrimary;
+        this.$vuetify.theme.background = value ? this.$vuetify.theme.backgroundDark : this.$vuetify.theme.backgroundLight;
+        this.$vuetify.theme.background2 = value ? this.$vuetify.theme.backgroundDark : this.$vuetify.theme.background2Light;
+        this.$vuetify.theme.search = value ? this.$vuetify.theme.backgroundDark : this.$vuetify.theme.searchLight;
+        this.$vuetify.theme.crlogo = value ? this.$vuetify.theme.crlogoDark : this.$vuetify.theme.crlogoLight;
+      }
```

Comment:
I'm pretty sure this whole block can be handled just by setting different themes for dark and light mode in the initial theme setup